tests/poll/delete.yml

```diff
@@ -0,0 +1,122 @@
+---
+db:
+  poll/1/meeting_id: 1
+
+
+cases:
+- name: poll
+  action: poll.delete
+  payload:
+    id: 1
+
+  cases:
+  - name: motion
+    db:
+      poll/1/content_object_id: motion/2
+    cases:
+    - name: Without perm
+      is_allowed: false
+
+    - name: With perm
+      permission: motion.can_manage
+      is_allowed: true
+
+  - name: assignment
+    db:
+      poll/1/content_object_id: assignment/2
+    cases:
+    - name: Without perm
+      is_allowed: false
+
+    - name: With perm
+      permission: assignment.can_manage
+      is_allowed: true
+
+  - name: other
+    cases:
+    - name: Without perm
+      is_allowed: false
+
+    - name: With perm
+      permission: agenda_item.can_manage
+      is_allowed: true
+
+- name: option
+  action: option.delete
```

Comment:
Same as for `vote.delete`. There is no `option.delete` action.

Reply:
Also see the script `missing.py` in this repo. If you remove an action from the backend, the python script would tell it.